.github/workflows/pr-merge.yml

```diff
@@ -23,3 +23,6 @@ jobs:
   metrics:
     needs: [ ci-job ]
     uses: ./.github/workflows/report.yml
+
```

Comment:
As tests have been moved out from `checks.yml`, then secrets no longer need to be inherited to checks.

Reply:
On the other hand: As tests have been moved out from `checks.yml`, then pr-merge maybe needs to execute tests individually?

Reply:
Also the name of `ci-job` should maybe changed to "checks", then?